### PR TITLE
Update cards, tiles, shelf and grid

### PR DESCRIPTION
**Pull request by thomasthesecond**
- Tiles weren't fully responsive
- Moved responsive styles out of CardVideo into "global" card components
- Use `30px` gutter for shelf and grid
- Change width of shelf cards and grid tiles from `412px` to `410px`, and `290px` to `300px` for `30px` gutters and `1290px` grid
- Create propTypes.js utility
- Replace CardDescription and CardDescriptionPoster components with CardHeader and CardBullets; repurpose CardDescription component

### CODE REVIEW

File: src/utils/propTypes.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { PropTypes } from "react";
```
Comment (nickfranciosi) [inline src/utils/propTypes.js]: this is great, I hate having to look up what the props should be for styles every time 👍

**Comment by craig-murray** [comment]
@thomasthesecond how can I review this work?

**Comment by thomasthesecond** [comment]
http://lonelyplanet.github.io/backpack-ui/

**Comment by craig-murray** [comment]
Reviwed this fix at http://lonelyplanet.github.io/backpack-ui/?knob-Title=The%20shop&knob-Layout=card&knob-Image%20path=https%3A%2F%2Flonelyplanetstatic.imgix.net%2Fcopilot%252Fimages%252FYXJ0YW5kY3VsdHVyZS5qcGdTYXQgRGVjIDE3IDIwMTYgMjE6MDA6MDUgR01UKzAwMDAgKFVUQyk%253D.jpg%3Fq%3D60%26sharp%3D10%26fit%3Dcrop%26w%3D180&knob-Description%5B0%5D=Item%201&knob-Description%5B1%5D=Item%202&knob-Heading=High%20Sierra%20routes%20with%20Ken%20Walker%20Smith&knob-Theme=light&knob-Year=2016&knob-Image%20source=%2F%2Fmedia.gadventures.com%2Fmedia-server%2Fcache%2Fa6%2F2c%2Fa62ca9f86982dd950319138334e7248b.jpg&knob-URL=%23&knob-Bullets%5B0%5D=On%20The%20Road&knob-Bullets%5B1%5D=E.01&knob-Icon%20name=Watch%20later&knob-Video%20runtime=129365&selectedKind=Video%20tile%20grid&selectedStory=Swiper&full=0&down=1&left=1&panelRight=0&downPanel=kadirahq%2Fstorybook-addon-actions%2Factions-panel

thx @thomasthesecond

**Comment by craig-murray** [comment]
Looks great!

**Pull request merged**